composer.json

```diff
@@ -37,23 +39,40 @@
         "psr/log": "^1.0 || ^2.0",
         "symfony/config": "^2.1 || ^3.0 || ^4.0 || ^5.0 || ^6.0",
         "symfony/console": "^2.1 || ^3.0 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/stopwatch": "^2.0 || ^3.0 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/yaml": "^2.0.5 || ^3.0 || ^4.0 || ^5.0 || ^6.0"
+        "symfony/stopwatch": "^2.1 || ^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/yaml": "^2.1 || ^3.0 || ^4.0 || ^5.0 || ^6.0"
     },
+
     "require-dev": {
         "phpunit/phpunit": "^4.8.35 || ^5.4.3 || ^6.0 || ^7.0 || ^8.0 || ^9.0",
         "sanmai/phpunit-legacy-adapter": "^6.1 || ^8.0"
     },
+
     "suggest": {
+        "friendsofphp/php-cs-fixer": "Install globally for check and fix codestyle",
```

Comment:
to whoever is using PHP Coveralls, they will not have any extra value for Coveralls integration when installing the Fixer as well. Those are 2 separated projects

Reply:
I will remove